assets/src/edit-story/components/library/panes/text/textPresets.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { PAGE_WIDTH } from '../../../../constants';
+import { dataFontEm } from '../../../../units';
+import { TEXT_ELEMENT_DEFAULT_FONT } from '../../../../app/font/defaultFonts';
+
+// By default, the element should be 50% of the page.
+const DEFAULT_ELEMENT_WIDTH = PAGE_WIDTH / 2;
+
+const DEFAULT_PRESET = {
+  id: 'default-text',
+  content: `<span style="font-weight: 400">${__(
+    'Fill in some text',
+    'web-stories'
+  )}</span>`,
+  fontSize: dataFontEm(1),
+  font: TEXT_ELEMENT_DEFAULT_FONT,
+  width: 160, // Approximately spans the text inside it.
+};
+
+const PRESETS = [
+  {
+    id: 'heading',
+    title: __('Heading', 'web-stories'),
+    content: `<span style="font-weight: 700">${__(
+      'Heading',
+      'web-stories'
+    )}</span>`,
+    fontSize: dataFontEm(2),
+    font: {
```

Comment:
This is not nearly enough data, unfortunately, to properly display/render the text element. See `TEXT_ELEMENT_DEFAULT_FONT` for what that usually requires. Can't we use `TEXT_ELEMENT_DEFAULT_FONT` here as well?

Reply:
Please use `TEXT_ELEMENT_DEFAULT_FONT` in place of "Open Sans" everywhere in this file.

Reply:
Done.